site/docs/resources/comparison.md

```diff
@@ -150,6 +150,11 @@ As a natively compiled language, Go outperforms TypeScript at raw CPU speed by s
 This is much less relevant when you write a bot because most of the time is spent waiting for the network, but eventually, it will start to matter how fast your bot can parse JSON.
 Go can be a better choice in these cases.
 
+Another metric is developer experience (DX).
+In general, Go and TypeScript are both known to have extremely good tooling and editor support.
+However, for the Bot API in particular, grammY is significantly better than any library written in Go.
```

Comment:
Why not specify a reason?

Reply:
Maybe something like "because of TypeScript"?

Reply:
True, I'll add that